examples/great_expecations/README.md

```diff
@@ -0,0 +1,20 @@
+## Data Profiler + Great Expectation
+[Great Expectations](https://greatexpectations.io/) is an open source tool built around the idea that should always know what to expect from your data. It helps data teams eliminate pipeline debt, through data testing, documentation, and profiling.
+
```

Comment:
we didn't add the `About these Notebooks` but let's put a marker for that in the next PR @micdavis CC @JGSweets